website/server/models/group.js

```diff
@@ -392,7 +392,18 @@ schema.methods.sendChat = function sendChat (message, user) {
   let newMessage = chatDefaults(message, user);
 
   this.chat.unshift(newMessage);
-  this.chat.splice(200);
+
+  const MAX_CHAT_COUNT = 200;
+  const MAX_SUBBED_GROUP_CHAT_COUNT = 400;
+
+  let maxCount = MAX_CHAT_COUNT;
+
+  let plan = this.purchased.plan;
+  if (plan && plan.customerId && !plan.dateTerminated) {
```

Comment:
So the active sub check should be:

`if (plan && plan.customerId && (!plan.dateTerminated || plan.dateTerminated < now)) {`